src/providers/LayerProvider/LayerProvider.tsx

```diff
@@ -348,7 +345,7 @@ export const LayerProvider = ({
   const drawerContextData = useDrawer()
 
   return (
-    <SWRConfig value={defaultSWRConfig}>
+    <SWRConfig value={{ ...defaultSWRConfig, provider: () => new Map() }}>
```

Comment:
Every time we re-initialize this `<SWRConfig`, we'll reset the cache. That means if you unmount and remount the component, you don't need to actually call `expireCaches`, we'll just automatically refresh everything.